ee/server/service/hosts.go

```diff
@@ -0,0 +1,15 @@
+package service
+
+import (
+	"context"
+
+	"github.com/fleetdm/fleet/v4/server/fleet"
+)
+
+func (svc *Service) GetHost(ctx context.Context, id uint, includeCVEScores bool) (*fleet.HostDetail, error) {
+	return svc.Service.GetHost(ctx, id, true)
```

Comment:
Shouldn't this be `return svc.Service.GetHost(ctx, identifier, includeCVEScores)` ?

Reply:
This is a bit odd, but it does do the right thing. The issue is there aren't multiple implementations of `getHostEndpoint`, only the one in `server/service/hosts.go`. It can't know whether `includeCVEScores` should be `true` or `false`, so it just passes `false`. In the version of `GetHost` in `ee/server/service/hosts.go`, we set it to `true`.

Reply:
Should we add a comment about it?

Reply:
Would something like `// include cve scores in premium` suffice? Assume this applies to all the other methods in `ee/server/service`

Reply:
Or `// reuse Service.GetHost but forcing includeCVEScores to true since they are included in premium` possibly.